click/js/app.js

```diff
@@ -1,5 +1,12 @@
-console.log("Sanity Check: JS is working!");
+//1.READY
+alert("Go!");
 
 $(document).ready(function() {
 
 });
+
+//3.CLICK
+$("span").on("click", function createList(event){
```

Comment:
Make sure this code goes inside the `$(document).ready` above. Otherwise the click listener won't have any elements to bind to (because the html document isn't fully loaded when this code is executed).

Also, let's do something about `something`. Can you think of a a more "semantic" variable name -- something that describes what it is and even suggests its datatype? `text_content`, `title`, `label`?